app/javascript/components/Pages/EmotionIntensity.js

```diff
@@ -1,51 +1,162 @@
-import React, {Fragment} from 'react';
+import React, {Fragment, useEffect, useState} from 'react';
 import {Footer, Header, Wrapper} from "../UI/ShareContent";
 import {capitalizeFirstLetter, isBlank} from "../helpers/helpers";
 import ButtonEmotion from "../UI/ButtonEmotion";
+import PoweredBy from '../../../assets/images/PoweredBy.svg';
 
+const IntenseLine = ({rating, setRating, comment, setComment, generateStyles, category, isBlankGif}) => {
+  const [isTextareaActive, setIsTextareaActive] = useState(false);
+  const handleRatingClick = (value) => setRating(value);
+  const handleCommentClick = (event) => {
+    setComment(event.target.value);
+    setIsTextareaActive(true);
+  };
+  const handleCommentFocus = (event) => {
+    if (!isTextareaActive) {
+      event.target.placeholder = '';
+      setIsTextareaActive(true);
+    }
+  };
+  return (
+  <div className="rating-comment-container">
+    <form>
+      <div className="form-group">
+        {[1, 2, 3, 4, 5].map((value) => (
+          <label key={value} className="rating-label" style={generateStyles(value, rating === value, category)}>
+            <input
+              type="radio"
+              name="rating"
+              value={value}
+              checked={rating === value}
+              onChange={() => handleRatingClick(value)}
+            />
+            {value}{value !== 1}
```

Comment:
are you sure you need **{value !== 1}** here?

Reply:
yes, expression {value !== 1}  is not necessary